doc/nrf/ug_nrf_cloud.rst

```diff
@@ -87,12 +87,16 @@ A device can successfully connect to `nRF Cloud`_ using MQTT if the following re
 
 * Preconnect provisioning
 
-  Run the  :file:`device_credentials_installer.py` Python script.
-  You need to specify a number of parameters including the device ID.
+  1. Run the `device_credentials_installer.py`_ Python script to create and install credentials on the device.
 
-  The script instructs the device to securely generate and store a private key, which never leaves the device.
-  It programs  the specified CA certificate into the device.
-  The private key never leaves the device, which makes this a more secure option.
+     You need to specify a number of parameters including the device ID.
+     The script instructs the device to securely generate and store a private key, which never leaves the device.
+     It programs  the specified CA certificate into the device.
```

Comment:
The CA that is specified by the `--ca` parameter is the the user's CA which is used to sign the device certificate.
The CA that is written to the device is the AWS CA.
So perhaps this should say something like:
_"The script creates a device certificate and signs it with the specified CA.  It writes the device certificate and AWS CA certificate to the device."_